src/sql/workbench/parts/query/browser/queryResultsView.ts

```diff
@@ -223,6 +223,11 @@ export class QueryResultsView extends Disposable {
 			this.input.state.visibleTabs = new Set();
 			this.input.state.activeTab = this.resultsTab.identifier;
 		}));
+		this.runnerDisposables.push(runner.onMessage(e => {
+			if (e.isError) {
```

Comment:
the one challenge I see with this is that focus may be switched to messages more than once in long running scripts.  For example, in the below scenario:

1. Execute a script with many batches that will run several minutes
2. First error occurs and focus goes to Messages 
3. User reads error and manually switches to results
4. User is in context looking at results
5. Second error occurs and focus goes to Messages

In the above scenario, ideally only the first error would switch focus to Messages.  This isn't a huge deal, so I'm fine with it as-is.

Reply:
After reviewing ssms some more, it appears it doesn't focus the messages until the query completes if there is an error message. Should we modify to be this behavior?